cvss.js

```diff
@@ -250,7 +250,16 @@ var CVSS = function (id, options) {
     l.appendChild(this.vector = e('a'));
     this.vector.className = 'vector';
     this.vector.innerHTML = 'CVSS:3.1/AV:_/AC:_/PR:_/UI:_/S:_/C:_/I:_/A:_';
-    
+    // setup the copy button/icon
+    l.appendChild(this.copyButton = e('button'))
+    this.copyButton.style.visibility = "hidden"
+    this.copyButton.className = "copyButton"
+    this.copyButton.innerHTML = '<img src="copy-icon.svg"/>'
```

Comment:
could you add an `alt` attribute to describe the icon? so those using screen readers will get it.